src/components/WwwFrame/PromotionalBanner/Banners/DepositIncentiveBanner.vue

```diff
@@ -27,15 +40,24 @@ export default {
 		return {
 			amountToLend: 0,
 			isLoggedin: false,
+			basketTotal: 0,
 		};
 	},
 	inject: ['apollo', 'cookieStore'],
 	apollo: {
 		query: amountToLendQuery,
 		preFetch: true,
+		variables() {
+			return {
+				basketId: this.cookieStore.get('kvbskt')
```

Comment:
The basket id variable is added to all queries automatically, so this isn't needed.